DomClasses/circle.js

```diff
@@ -0,0 +1,17 @@
+import { createCircle } from "./domInteractions.js"
+import { randomIntFromInterval } from "../Helpers/mathHelper.js"
+
+export class Circle {
+    constructor(widthBounceDiv,heightBounceDiv,leftBounceDiv,topBounceDiv){
+        this.circleRadius = randomIntFromInterval(40,100)
+        this.dom = createCircle(this.circleRadius)
```

Comment:
The `createCircle` function looks weird at first glance because we are in the constructor of the Circle class which has the role to create the circle. You need to split the functionality of the `createCircle` function into `createCircleDOMElement` and `addCircleToDOM`.

Reply:
This is not fixed yet.